pages/api/transactions/index.js

```diff
@@ -8,6 +8,19 @@ export default async function handler(request, response) {
       const transactions = await Transaction.find();
       return response.status(200).json(transactions);
     }
+
+    if (request.method === "POST") {
+      try {
+        const transactionData = request.body;
+        const transaction = new Transaction(transactionData);
+        await transaction.save();
```

Comment:
interesting way to use the Model. Is there a specific reason why you don't use our standard way? If not I would suggest to keep it coherent with other groups. This will help others to understand your code easier. 
Further, in the other api endpoints you switch to our standard way to interacting with collections (methods on the model). So changing this also makes your code more consistent which is important.